index.html

```diff
@@ -161,6 +172,40 @@ <h3>Skills</h3>
                 </ul>
             </div>
         </section>
+        <section class="contact-form">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/xayvgeok" method="post">
+                <ul>
+                    <li>
+                        <label for="name"></label>
+                        <input type="text" name="full-name" id="name" placeholder="Full name" required>
```

Comment:
- Please, kindly add the maxlength attribute to the name input to meet the project requirement.

Reply:
*README FILE*

- Please, kindly update the getting started header and add some steps on how users can have your repo running locally on their machine, and remove the unused headers added in the readme to make your readme more professional.

Reply:
@zieeco thanks alot to mention that i solved these problems